selenium/t/test_publish_source.py

```diff
@@ -34,11 +34,9 @@ def test_publish_source(self, connect_url):
         """Publish a document with source
         """
         pf = PublishContentForm()
-        pf.api_key.should(be.visible)
```

Comment:
Should we be asserting this in the Add Server dialog?

Reply:
Selene automatically requires that elements it tries to interact with are visible, so it's redundant.